tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Set Up Private Endpoint Connections
+summary: Learn how to set up private endpoint connections in TiDB Cloud.
+---
+
+# Set Up Private Endpoint Connections
+
+TiDB Cloud supports highy secure and one-way access to the TiDB Cloud service hosted in an AWS VPC via the [AWS PrivateLink](https://aws.amazon.com/privatelink/?privatelink-blogs.sort-by=item.additionalFields.createdDate&privatelink-blogs.sort-order=desc), as if the service were in your own VPC. A private endpoint is exposed in your VPC and you can create a connection to the TiDB Cloud service via the endpoint with permission.
+
+Powered by AWS PrivateLink, the endpoint connection is secure and private, and does not expose your data to the public internet.
+
+The architecture of the private endpoint is as follows:
+
+![Private endpoint architecture](/media/tidb-cloud/aws-private-endpoint-arch.png)
+
+The cloud vendor AWS generates a specific DNS hostname for your endpoint service. Before you grant permissions to service consumers or specific AWS principals (AWS accounts, IAM users, and IAM roles), they do not have access to your endpoint service. For more detailed definitions of the private endpoint and endpoint service, see the following AWS documents:
+
+- [What is AWS PrivateLink?](https://docs.aws.amazon.com/vpc/latest/privatelink/what-is-privatelink.html)
+- [AWS PrivateLink concepts](https://docs.aws.amazon.com/vpc/latest/privatelink/concepts.html)
+
+The relationship of a TiDB cluster in TiDB Cloud, an endpoint service, and a private endpoint is as follows:
+
+- Each TiDB cluster corresponds to one endpoint service, and each endpoint service corresponds to one TiDB cluster.
+- Each endpoint service corresponds to one or multiple private endpoints.
+- Each TiDB cluster corresponds to one or multiple private endpoints.
+
+> **Note:**
+>
+> - Currently, TiDB Cloud supports private endpoint connection only when the endpoint service is hosted in AWS. If the service is hosted in Google Cloud Platform (GCP), the private endpoint is not applicable.
+> - The private endpoint support is provided only for the TiDB Cloud Dedicated Tier, not for the Developer Tier.
+> - Private endpoint connection across regions is not supported.
+
+## Why should I use private endpoint
```

Comment:
move to the overview of private endpoint

Reply:
done